python/ccxt/okcoinusd.py

```diff
@@ -128,6 +128,12 @@ def describe(self):
                     'https://www.npmjs.com/package/okcoin.com',
                 ],
             },
+            'fees': {
+                'trading': {
+                    'taker': 0.002,
+                    'maker': 0.002,
+                }
+            }
```

Comment:
Hi! Thanks for you contribution! We really appreciate it! Can you please move the hitbtc2.py / okcoin.py changes to hitbtc2.js and okcoin.js, respectively? ) The exchange.py is fine! Thx again! 

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code

Reply:
Ok, but unfortunately i dont have the time to check(mainly to learn and setup the env) transpilation.
Is the automatic build system checking that?

Reply:
@beniloew yes, it does, don't worry too much, I will edit it where necessary, if you move the edits to js ) No need to install everything, just copy the formatting from around there. Well, you know. Thx again!